17/umbraco-commerce/key-concepts/product-adapters.md

```diff
@@ -71,6 +71,11 @@ public interface IProductSnapshot
 
 ```
 
+{% hint style="info" %}
+`IProductSnapshot` is the base model for what Umbraco Commerce needs. If you want to include images or measurements (for example, if you need weight-based shipping), then there are other interfaces you can return.
+`IProductSnapshotWithImage`, `IProductSnapshotWithCategories` & `IProductSnapshotWithMeasurements` are all valid return types that allows you to enrich the product model.
```

Comment:
Not sure if there are in fact more available interfaces - you probably know that better than me ;)
But how about we use a list here? 🤔 

```suggestion
Valid interfaces include, but are not limited to:

* `IProductSnapshotWithImage`
* `IProductSnapshotWithCategories`
* `IProductSnapshotWithMeasurements`
```

Reply:
These are all the ones I could find, which also makes mentioning it being not limited to them a bit misleading 🙂